AK/bits/Math/libm_fallback.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2021, Leon Albrecht <leon2002.la@gmail.com>.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "./common.h"
```

Comment:
Since we explicitly expect Lagom and possibly non-serenity project owned users to eventually include this file, the relative path "./common.h" include seems potentially dangerous if the project trying to use Lagom has its own "common.h". Can we use the full path here? `<AK/bits/Math/common.h>`.